tests/IBFE/instrument_panel_01.cpp

```diff
@@ -583,5 +600,37 @@ main(int argc, char** argv)
             dX_vec = 0.0;
         }
 
+        // fourth test: translate and add a background velocity
+        {
+            plog << "\ntest 3\n";
+            FEDataManager* fe_data_manager = ib_method_ops->getFEDataManager();
+            EquationSystems* equation_systems = fe_data_manager->getEquationSystems();
+            System& dX_system = equation_systems->get_system(IBFEMethod::COORD_MAPPING_SYSTEM_NAME);
+            NumericVector<double>& dX_vec = *dX_system.solution;
+            dX_vec = 3;
+            System& X_system = equation_systems->get_system(IBFEMethod::COORDS_SYSTEM_NAME);
+            NumericVector<double>& X_vec = *X_system.solution;
+            X_vec += dX_vec;
+            System& U_system = equation_systems->get_system(IBFEMethod::VELOCITY_SYSTEM_NAME);
+            NumericVector<double>& U_vec = *U_system.solution;
+            std::vector<dof_id_type> component_dofs;
+            for (auto node_it = mesh.nodes_begin(); node_it != end_node_it; ++node_it)
+            {
+                Node& node = **node_it;
+                for (int d = 0; d < NDIM; ++d)
+                {
+                    IBTK::get_nodal_dof_indices(U_system.get_dof_map(), &node, d, component_dofs);
+                    TBOX_ASSERT(component_dofs.size() == 1);
+                    // NOTE: velocity needs to be evaluated using the shifted coordinates!
+                    U_vec.set(component_dofs[0], (d + 1) * (node(d) + 3));
```

Comment:
Yeah, I realized that my mistake as using `interpolateVelocity` which does not set values of the solution - this looks good.